### PR TITLE
Homepage  Consolidation

### PR DESCRIPTION
**Pull request by Mitso**
Added surveys and comments to Hero article.

### CODE REVIEW

**Comment by Mitso** [comment]
@codiebeulaine Please review 

screencapture-localhost-8000-2019-05-28-17_27_56

File: gem/templates/chhaajaa/base.html [hunk 1]
```diff
@@ -94,7 +108,9 @@
               <a href="{{VIEW_PROFILE_URL}}" class="navbar-nav__anchor"><span id="logout">{% trans "Profile" %}</span></a>
             </li>
             <li class="navbar-nav__item">
-              <a href="javascript::void(0)" id="logout-btn" class="navbar-nav__anchor"><span id="logout">{% trans 'Log out' %}</span></a>
+              <a href="javascript::void(0)" id="logout-btn" class="navbar-nav__anchor">
```
Comment (codiebeulaine) [inline gem/templates/chhaajaa/base.html]: should this Js void be here?

File: gem/templates/chhaajaa/surveys/surveys_list.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+            {% if not survey.your_words_competition and not survey.display_survey_directly %}
+            <div class="surveys surveys--profiles-join">
+              <div class="surveys__wrapper">
+                <div class="heading heading heading--secondary">
+                  <h1 class="heading__title">{% trans 'Quiz' %}</h1>
+                </div>
+                <div class="surveys__inner">
+                  <section>
+                    <h1 class="heading__headline">
+                      {% trans "Share your opinions and stories, take polls, win fun prizes." %}
```
Comment (codiebeulaine) [inline gem/templates/chhaajaa/surveys/surveys_list.html]: some of this Springster text will have to be taken out, will it be done so at a later stage?

Comment (Mitso) [reply gem/templates/chhaajaa/surveys/surveys_list.html]: @codiebeulaine 
Updated to generic text.

**Pull request merged**